src/Microsoft.DotNet.Cli.Utils/Command.cs

```diff
@@ -181,13 +177,15 @@ public ICommand EnvironmentVariable(string name, string value)
         public ICommand CaptureStdOut()
         {
             ThrowIfRunning();
+            EnsureStdOut();
             _stdOut.Capture();
             return this;
         }
 
         public ICommand CaptureStdErr()
         {
             ThrowIfRunning();
+            EnsureStdErr();
```

Comment:
No there isn't.  The idea being that Ensure... is only called if the caller explicitly asks for Capture/Forward/etc.  Since there are so many of these public methods, these calls need to be in each of these public methods.